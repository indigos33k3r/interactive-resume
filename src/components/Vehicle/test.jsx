import React from 'react'
import {shallow} from 'enzyme'

describe('Vehicle', () => {
  it('renders <Vehicle />', () => {
    const component = render()

    expect(component).toMatchSnapshot()
  })

  function render() {
    const defaultProps = {
      vehicleClass: 'batmobil'
    }
    const Vehicle = require('.').default

    return shallow(<Vehicle {...defaultProps} />)
  }
})
