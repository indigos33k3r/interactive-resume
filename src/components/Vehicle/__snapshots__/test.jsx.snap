// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Vehicle renders <Vehicle /> 1`] = `
<div
  className="vehicle vehicle--batmobil"
/>
`;
